spec/features/admin/sign_in_as_admin_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'features/support/sign_up_helpers'
+require 'features/support/not_signed_in'
+
+describe 'signing as an admin user' do
+  let!(:user) { create(:user, :confirmed, admin: true) }
+
+  context 'when visiting the home page' do
+    before do
+      5.times { create(:organisation) }
+      sign_in_user user
+      visit root_path
+    end
+
+    it 'redirects to the organisations page' do
+      expect(page.current_path).to eq(organisations_path)
+    end
+
+    it 'shows list of signed organisations on the home page' do
+      Organisation.all.each do |organisation|
```

Comment:
This is also checking for the organisation that the super admin user belongs to, which only passes as its displayed top left in the subnav. This spec is specifically looking at the table though.

Reply:
Actually I'm wrong, this will show all organisations as the super admins organisation will be one of the signed up organisations.